### DISCUSSION TITLE
Flysky is using clone of Stm32 inside their new i6x transmitters

### DISCUSSION THREAD

**Discussion by jaytxrx** [OpenI6X/opentx#333] [General]
I was planning to buy a new controller due to the active development happening in this community. 

Recently I noticed that the AliExpress seller mentioning the clone chip. Instead of  STM32F072VBT6, they are using now APM32F072VBT6.

By the way, does anyone have a controller with the clone chip and tried flashing opentx ?

See below
https://aliexpress.com/item/1005003052731317.html

![image](https://user-images.githubusercontent.com/431506/211092799-ce4bb901-f76f-4ed1-b71e-cd2ccd0e8ee7.png)

**Comment by psicho100pro** [comment]
https://www.rcgroups.com/forums/showpost.php?p=48731989&postcount=227

**Comment by jaytxrx** [comment]
Good to know that it works
rcgroups.com/forums/showthread.php?3916435-FlySky-I6X-port-of-OpenTX/page16

**Reply by Ultraexpert** [reply]
No issues so far with the new chip.

**Comment by ajjjjjjjj** [comment]
You can also check our wiki, where APM32 is also mentioned:
https://github.com/OpenI6X/opentx/wiki
